cookbooks/fb_iptables/recipes/packages.rb

```diff
@@ -19,14 +19,16 @@
 # limitations under the License.
 #
 
-# c10s is opted out until EPEL 10 becomes available
-if (node.centos? && !(node.centos7? || node.centos8? || node.centos10?)) || node.fedora?
+if node.centos_min_version?(9) || node.fedora?
   packages = ['iptables-legacy']
 else
   packages = ['iptables']
 end
+
 if node.ubuntu?
   packages << 'iptables-persistent'
+elsif node.centos_min_version?(9) || node.fedora?
+  packages << 'iptables-nft-services'
```

Comment:
This doesn't seem right -- if we're using legacy on >= 9 we shouldn't use the nft services there. This cookbook probably needs an overall knob to do stock/nft vs legacy/not-nft. @michel-slm what do you think?

Reply:
AFAICT, iptables-nft-servers *replaces* iptables-services. It literally `obsoletes` it, and on a fresh install of c9 or c10 you cannot install iptables-services, even if iptables-legacy is installed.

You can't even `dnf swap` them.

Reply:
(also as it stands this doesn't support stock nft at all)

Reply:
These files exist in ... both packages on both OSes.
```
/usr/libexec/initscripts/legacy-actions/ip6tables
/usr/libexec/initscripts/legacy-actions/ip6tables/panic
/usr/libexec/initscripts/legacy-actions/ip6tables/save
/usr/libexec/initscripts/legacy-actions/iptables
/usr/libexec/initscripts/legacy-actions/iptables/panic
/usr/libexec/initscripts/legacy-actions/iptables/save
```

Check this out - they're the same:

```
[root@scale-lists-centos9 ~]# rpm -qf /usr/libexec/initscripts/legacy-actions/iptables/panic
iptables-services-1.8.10-11.1.el9.noarch
iptables-nft-services-1.8.10-11.el9.noarch
[root@scale-lists-centos9 ~]# rpm -qV iptables-nft-services | grep legacy
[root@scale-lists-centos9 ~]# rpm -qV iptables-services | grep legacy
[root@scale-lists-centos9 ~]# 

```

Reply:
note that Fedora does not even have nft-services https://src.fedoraproject.org/rpms/iptables/blob/rawhide/f/iptables.spec

Reply:
@michel-slm - I think you're missing the point. I'm not supporting nft.

As it stands iptables-nft-services has *THE EXACT SAME FILES* in it as iptables-services, and if you install a C10 box with a kernel that supports good-old-iptables, and you try to install iptables-services **dnf will not let you**.

You **must** use `iptables-nft-services` with `iptables-legacy` to use LEGACY iptables.

Reply:
ok, so per @jaymzh iptables-nft-services can actually manage the legacy backend too, so we should at least install that in c10s. Don't touch Fedora though, on Fedora the package is still called iptables-services (don't ask) - there is no iptables-nft-services

I'll help check tonight if the legacy backend even works on the stock kernel, I don't have my baremetal c10s host right now